pkgs/development/python-modules/gym/default.nix

```
@@ -14,8 +14,10 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace setup.py \
-      --replace "pyglet>=1.2.0,<=1.3.2" "pyglet"
+      --replace "pyglet>=1.2.0,<=1.3.2" "pyglet" \
+      --replace "cloudpickle>=1.2.0,<1.4.0" "cloudpickle>=1.2.0,<1.7.0"
```

so we don't have to revisit this in the immediate future
```suggestion
      --replace "cloudpickle>=1.2.0,<1.4.0" "cloudpickle~=1.2"
```

---

Done